Library/Formula/mediatomb.rb

```diff
@@ -1,31 +1,32 @@
-require 'formula'
-
 class Mediatomb < Formula
   desc "Open source (GPL) UPnP MediaServer"
-  homepage 'http://mediatomb.cc'
-  url 'https://downloads.sourceforge.net/mediatomb/mediatomb-0.12.1.tar.gz'
-  sha1 '86e880584cc9c8aaf3926d56048510d1d06e76b4'
+  homepage "http://mediatomb.cc"
+  url "https://downloads.sourceforge.net/mediatomb/mediatomb-0.12.1.tar.gz"
+  sha256 "31163c34a7b9d1c9735181737cb31306f29f1f2a0335fb4f53ecccf8f62f11cd"
+  revision 1
   bottle do
     sha1 "716145891055b842417844efb1f786344ba79f56" => :mavericks
     sha1 "d5fd8a1389771a2bf4677dc5f008b0c4249344aa" => :mountain_lion
     sha1 "ae3e70647489adec58c1433652d05bad3844f16a" => :lion
   end
 
-  revision 1
+  # error: explicit specialization of 'mutex' after instantiation
+  # SINGLETON_MUTEX(Timer, true);
+  depends_on MaximumMacOSRequirement => :mountain_lion
```

Comment:
Can you produce some logs for this? It build on Mavericks as recently as last November.